src/main/resources/regression/msmarco-v2-passage-unicoil-0shot.yaml

```diff
@@ -76,3 +76,35 @@ models:
       R@1000:
         - 0.7616
         - 0.7671
+  - name: rm3
```

Comment:
Maybe unicoil-0shot+rm3 is a better name?

Reply:
Fixed!